### PR TITLE
DE32879: Remove the hover class from a tile when animation completes.…

### PR DESCRIPTION
**Pull request by njostonehouse**
… Fixes the z-location of animated tiles, which were holding onto hover unnecessarily.

### CODE REVIEW

**Comment by njostonehouse** [comment]
Although `unpin` was removed when animation was complete in a previous commit, we also need to remove `hover` to restore the correct z index to the animated tiles.

**Comment by svanherk** [comment]
I may have messed up my BSI build, but I'm still seeing this as an issue.  Let me try to rebuild

**Comment by svanherk** [comment]
I'm seeing the `hover` effect be removed from the new tile sliding into place, but if I hover back over it and move away, it stays and never turns off.

**Comment by njostonehouse** [comment]
> I'm seeing the `hover` effect be removed from the new tile sliding into place, but if I hover back over it and move away, it stays and never turns off.

Well that's charming. I'll look into it.

File: src/d2l-course-tile.js [hunk 1]
```diff
@@ -570,6 +570,8 @@ Polymer({
 		}
 
 		this.toggleClass('unpin', false, this);
+		this._pinAnimationInProgress = false;
```
Comment (njostonehouse) [inline src/d2l-course-tile.js]: Before the tile-reuse problem came into play, this didn't need to be explicitly set to `false` when the animation finished. Now, it stays `true` and causes problems for things like removing the `.hover` styles.

**Pull request merged**